packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -419,6 +422,14 @@ export default defineComponent({
 		},
 		async handleSelect(key: string) {
 			switch (key) {
+				case 'templates':
+					if (
+						this.settingsStore.isTemplatesEnabled &&
```

Comment:
The sidebar option will be different for custom template host but it will have the same key. So this will prevent this event from being sent if users still have in-app templates (custom host)

Reply:
Check. Merge away